security/README.md

```diff
@@ -0,0 +1,35 @@
+# Security
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [RFC 2119](http://www.ietf.org/rfc/rfc2119.txt).
+
+# Services
+
+Services SHOULD check client [OAuth scopes](../security/oauth.md) and [OpenID Connect claims](http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) (e.g. `sub` ) for proper authorization.
+
+Services SHOULD be on a private network inaccessible from the Internet whenever possible.
+
+Publicly exposed services SHOULD be behind an API Gateway and use the API Gateway for authentication, DoS protection, and rate limiting.
+
+Services SHOULD use HTTPS for connections.
+
+# Applications
+
+Applications MUST NOT expose private credentials to users or in source control (e.g. OAuth credentials, tokens, API keys, etc.).
+
+Applications SHOULD use environment variables to store private credentials.
+
+Front-end applications MUST NOT store any private credentials in browser cache or cookies.
+
+Applications SHOULD protect against [XSS and CSRF](http://www.redotheweb.com/2015/11/09/api-security.html) attacks.
+
+Applications SHOULD use HTTPS for connections.
+
+Applications SHOULD sanitize all user inputs, URL parameters or any input parameters exposed to the user to prevent attacks (e.g. XSS, SQL injection).
+
+When storing passwords, `bcrypt` SHOULD be used.
+
+# General Guidelines
+
+Two-factor authentication (2FA) SHOULD be enabled for account logins (e.g. AWS, Github, etc.)
```

Comment:
Thanks  @nodanaonlyzuul!

I added the `X-NYPL-Identity` guidance.

I think laptop encryption is a good idea. A MUST seems a little strong to me. Maybe RECOMMENDED?